### ISSUE TITLE
Windows are considered doors

### ISSUE THREAD

**Issue by syphernl** [jseidl/hass-magic_areas#53]
A window entity with `device_class: window` is being included in the `*_doors` groups.  No `*_windows` groups are being created.

```
HASS Version | 0.118.3
HASS Type | Home Assistant Core
Magic_Areas version | 1.1.0
```

**Comment by jseidl** [comment]
That shouldn't be happening. Can you post the attributes (from developer tools) of both the `area_door_*` and the device with `device_class: window` that is going into that aggregation?

**Comment by JakeHertenstein** [comment]
I've run into the same issue. I turned on debugging but no Window aggregates are created.
Attributes of my `binary_sensor.openclose_77` (a Sonoff DS01 contact switch over Deconz):
```
on: true
friendly_name: Bedroom Window
device_class: window
```

`binary_sensor.area_bedroom`:
```
presence_sensors: binary_sensor.bedroom_occupancy, binary_sensor.presence_25, binary_sensor.openclose_32, binary_sensor.openclose_34, binary_sensor.openclose_77, binary_sensor.withings_in_bed_jake, switch.area_presence_hold_bedroom
active_sensors: binary_sensor.openclose_32, binary_sensor.withings_in_bed_jake
lights: light.nightstands, light.dresser
climate: 
clear_timeout: 600
update_interval: 15
on_states: on, home, playing, open
exterior: false
automatic_lights: disabled
friendly_name: Area (Bedroom)
icon: mdi:texture-box
device_class: occupancy
```

Settings:
```
    bedroom:
      clear_timeout: 600
      control_lights: false
      control_climate: false
      include_entities:
        - binary_sensor.withings_in_bed_jake
      exclude_entities:
        - binary_sensor.master_bedroom_fan_occupancy
        - light.extended_color_light_41
        - light.extended_color_light_44
        - light.bedroom
```

Versions:
```
Home Assistant: 2021.1.1
Type: Supervised
magic_areas: v.1.1.2
```

**Comment by jseidl** [comment]
Customizing is not respected since it looks like it gets applied after MA is loaded. What device class does this sensor have if you don't customize it?

**Comment by JakeHertenstein** [comment]
I guess that makes sense. I think I ran into the same problem with a "switch" as I had to create a dummy "light" template to make that work... seems there's an opportunity for improvement here?

**Comment by jseidl** [comment]
Yes this is a known issue, currently investigating how to get around it.

**Comment by jseidl** [comment]
All `customize` related issues are being fixed with rework #74, coming up on release 2.0.0.

> I think I ran into the same problem with a "switch" as I had to create a dummy "light" template to make that work... seems there's an opportunity for improvement here?

Switches will not be automated by Magic Areas since they can be anything. The recommended approach for `switch` entities that control lights is to use https://www.home-assistant.io/integrations/light.switch/

**Comment by JakeHertenstein** [comment]
Ohh, that's a new one for me. Thank you!!

**Comment by jseidl** [comment]
Possibly fixed by #74, released on [2.0.0](https://github.com/jseidl/hass-magic_areas/releases/tag/v2.0.0).

Can you folks test it out and let me know if the issue is gone?
